issue-triaging.md

```diff
@@ -71,3 +71,55 @@ triaged and is ready for work. If someone who is assigned to an issue is no long
 | `Sponsor Needed`     | A new component has been proposed, but implementation is not ready to begin. This can be because a sponsor has not yet been decided, or because some details on the component still need to be decided.        |
 | `Accepted Component` | A sponsor has elected to take on a component and implementation is ready to begin.                                                                                                                             |
 | `Vendor Specific Component` | This should be applied to any component proposal where the functionality for the component is particular to a vendor.                                                                                          |
+
+### State diagram
+
+Here is a diagram outlining the potential issue states and how issues move through different stages:
+
+```mermaid
+flowchart TD
+  subgraph graph1["**needs-triage**"]
+        n1(["Needs Triage"])
+  end
+  subgraph graph2["**waiting-for-codeowners**"]
+        n3["Waiting for Codeowners<br>to further validate the issue"]
```

Comment:
I'm not sure if we want to add this part to the graph, but, what should the triager do if the issue becomes stale while waiting for codeowners?

Reply:
> I'm not sure if we want to add this part to the graph, but, what should the triager do if the issue becomes stale while waiting for codeowners?

To me this is a sign that there codeowners are not responsive. We should take reasonable steps to let them know, but otherwise this would (if severe enough) lead to the component being marked as unmaintained